packages/amazonq/test/e2e_new/amazonq/utils/generalUtils.ts

```diff
@@ -13,6 +13,31 @@ export async function sleep(timeout: number) {
     await new Promise((resolve) => setTimeout(resolve, timeout))
 }
 
+/**
+ * Minimal assertion library for testing
+ * @param actual The value to test
+ * @returns Object with assertion methods (equals, include, is.true)
+ */
+export const expect = (actual: any) => ({
```

Comment:
theres a build in library for this in node called `assert`.

Reply:
Its built into node itself, so you shouldn't need to add a dependency ([source](https://nodejs.org/api/assert.html)). Here's an [example](https://github.com/aws/aws-toolkit-vscode/blob/0769acb2ecd3636db62a43761f1393f1ec6a0dc5/packages/core/src/test/docdb/utils.test.ts#L11).